src/cli.rs

```diff
@@ -8,6 +8,21 @@ pub struct Args {
     /// Enabled by default if the app is running under `cargo leptos ...`
     #[arg(short = 'n', long = "no-open", default_value_t = in_cargo_leptos())]
     pub no_open: bool,
+    /// This flag alters the level log based on count of arg.
+    ///
+    /// 1 -> DEBUG, 2 -> TRACE, Default -> INFO
+    #[arg(short = 'v', long = "verbose", action = clap::ArgAction::Count , default_value_t = 0)]
+    pub verbose: u8,
+}
+
+impl Args {
+    pub fn log_level(&self) -> tracing::Level {
+        match self.verbose {
+            1 => tracing::Level::DEBUG,
+            2 => tracing::Level::TRACE,
+            _ => tracing::Level::INFO,
```

Comment:
What happens if the user passes, say, `-vvv`?

Reply:
Updated case match with 0, 1, _